public/app/plugins/datasource/loki/components/LokiContextUi.tsx

```diff
@@ -1,4 +1,5 @@
 import { css } from '@emotion/css';
+import { cloneDeep } from 'lodash';
```

Comment:
`structuredClone`?

Reply:
I wouldn't be surprised if at some point a "remove lodash" task is created for Grafana as a whole.

Reply:
Oh, TIL! Thanks so much!